browser/prefs/brave_pref_service_incognito_allowlist.cc

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/prefs/brave_pref_service_incognito_allowlist.h"
+
+#include "base/no_destructor.h"
+#include "brave/browser/ui/tabs/brave_tab_prefs.h"
```

Comment:
This including should be guarded with `!IS_ANDROID`?

Reply:
Done.